README.md

```diff
@@ -2,7 +2,7 @@
 
 ## About
 
-PeterPortal is a web application aimed to aid UCI students with course discovery and planning. We consolidate public data available on multiple UCI sources on the application to improve the user experience when planning their course schedule.
+PeterPortal is a web application aimed at aiding UCI students with course discovery and planning. We consolidate public data available on multiple UCI sources in the application to improve the user experience when planning course schedules.
```

Comment:
"designed to aid" is better than bot "aimed to aid" and "aimed at aiding" in my opinion

Reply:
Looks good!